src/Exceptions/UnresolvedPage.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DrewRoberts\Blog\Exceptions;
+
+use Exception;
+
+class UnresolvedPage extends Exception /*implements LocationException*/
```

Comment:
I doubt this is is needed.  If an url is entered and page doesn't exist, it will be a 404 error.  Again, location handling is a bit different.